psi4/src/psi4/libfunctional/LibXCfunctional.cc

```diff
@@ -227,50 +227,60 @@ void LibXCFunctional::set_tweak(std::vector<double> values) {
     if (xc_func_name_ == "XC_GGA_X_B86") {
         if (vsize == 3) {
             // (XC(func_type) *p, FLOAT beta, FLOAT gamma, FLOAT omega);
-            xc_gga_x_b86_set_params(xc_functional_.get(), values[0], values[1], values[2]);
+            // xc_gga_x_b86_set_params(xc_functional_.get(), values[0], values[1], values[2]);
+            xc_func_set_ext_params(xc_functional_.get(), values.data());
             failed = false;
         }
     } else if (xc_func_name_ == "XC_GGA_X_B88") {
         if (vsize == 2) {
             // (XC(func_type) *p, FLOAT beta, FLOAT gamma);
-            xc_gga_x_b88_set_params(xc_functional_.get(), values[0], values[1]);
+            // xc_gga_x_b88_set_params(xc_functional_.get(), values[0], values[1]);
+            xc_func_set_ext_params(xc_functional_.get(), values.data());
             failed = false;
         }
     } else if (xc_func_name_ == "XC_GGA_X_PBE") {
         if (vsize == 2) {
-            //  (XC(func_type) *p, FLOAT kappa, FLOAT mu);
-            xc_gga_x_pbe_set_params(xc_functional_.get(), values[0], values[1]);
+            // (XC(func_type) *p, FLOAT kappa, FLOAT mu);
+            // xc_gga_x_pbe_set_params(xc_functional_.get(), values[0], values[1]);
+            xc_func_set_ext_params(xc_functional_.get(), values.data());
             failed = false;
         }
     } else if (xc_func_name_ == "XC_GGA_C_PBE") {
-        if (vsize == 1) {
-            // (XC(func_type) *p, FLOAT beta);
-            xc_gga_c_pbe_set_params(xc_functional_.get(), values[0]);
+        if (vsize == 3) {
+            // (XC(func_type) *p, FLOAT beta); FLOAT gamma, FLOAT BB
+            // xc_gga_c_pbe_set_params(xc_functional_.get(), values[0]);
+            values[1] = xc_func_info_get_ext_params_default_value(const_cast<xc_func_info_type*>(xc_functional_->info), 1);
```

Comment:
Looking at this, at some point (not this PR) it may be good to allow input like `{"beta": 5}` as we can now obtain the name of the input parameters:
```
  {"_beta",  0.06672455060314922,     "beta constant"},
  {"_gamma", 0.031090690869654895034, "(1 - ln(2))/Pi^2 in the PBE"},
  {"_B",     1.0, "Multiplies the A t^2 term. Used in the SPBE functional"},
```

Reply:
Good notion.

The libxc change that alarmed me was if they're now setting omega and the other parameters in one command, are we going to run into trouble with separate omega and other tweakers?